src/components/HowRichAmI/index.js

```diff
@@ -501,73 +522,46 @@ MethodologyDialog.propTypes = {
   onClose: PropTypes.func.isRequired
 }
 
-const creditsStyles = theme => ({
-  root: {
-    margin: theme.spacing() * 2
-  }
-})
-const Credits = withStyles(creditsStyles)(({ classes }) => <div className={classes.root}>
-  <Typography>
-    The How Rich Am I Calculator is a project of <a href='https://www.givingwhatwecan.org'>Giving What We Can</a>
-  </Typography>
-</div>)
-
-Credits.propTypes = {
-  classes: PropTypes.object
-}
-
-const callToActionStyles = theme => ({
-  logoBackground: {
-    height: 130,
-    width: 130,
-    backgroundColor: theme.palette.primary.main,
-    margin: '0 auto',
-    display: 'flex',
-    alignItems: 'center',
-    justifyContent: 'center'
-  }
-})
+const MethodologyFooter = withStyles(methodologyDialogStyles)(({ onClick, classes }) => (
+  <div className={classes.footer}>
+    <Typography className={classes.footerText} onClick={onClick}>
+      Methodology and Data Sources
+    </Typography>
+  </div>
+))
 
-const CallToAction = withStyles(callToActionStyles)(({ classes }) => (
-  <Grid container spacing={GRID_SPACING} justify="center">
-    <Grid item xs={8}>
-      <Grid container spacing={GRID_SPACING}>
-        <Grid item xs={12}>
-          <img
-            src="https://d33wubrfki0l68.cloudfront.net/39338ba45f09702b3455361490262a6d873a66eb/fa985/images/logos/gwwc-logo-transparent-nav-2021.png"
-            alt="GWWC logo"
-          />
-        </Grid>
-        <Grid item xs={12}>
-          <Typography paragraph>
-            We{"'"}re a global community of people pledging to donate more, and
-            donate more effectively.
-          </Typography>
-        </Grid>
-        <Grid item xs={12} sm={6}>
-          <Button
-            href="https://www.givingwhatwecan.org"
-            color="secondary"
-            variant="contained"
-            fullWidth
-          >
-            Learn more
-          </Button>
-        </Grid>
-        <Grid item xs={12} sm={6}>
-          <Button
-            href="https://www.givingwhatwecan.org/pledge"
-            color="primary"
-            variant="contained"
-            fullWidth
-          >
-            Take the Pledge
-          </Button>
+const CallToAction = withSegment(withStyles(donationComparisonsStyles)(({ analytics, classes }) => {
+  const handleClick = useCallback(() => {
+    if (analytics) {
+      analytics.track("HRAI Explore Top Charities Clicked")
+    }
+  }, [analytics]);
```

Comment:
Looking into this I noticed that it works when using `window.analytics` instead of the analytics that is passed form withSegment. Not 100% sure what the root cause is but it seems to fix it...

Also: as it's pointing to another website, it might make sense to use the callback function instead. That way it delays it by some milliseconds which allows the segment tracking to complete first:

```suggestion
  const handleClick = async () => {
    const url = 'https://www.givingwhatwecan.org/best-charities-to-donate-to-2022'
    if (window.analytics) {
      await window.analytics.track('HRAI Explore Top Charities Clicked', {}, {}, () => {
        window.location.href = url
      })
    } else {
      window.location.href = url
    }
  }
```

```jsx
         <Button
              // No href here
              color="primary"
              variant="contained"
              size="large"
              onClick={handleClick}
            >
              Explore top charities
            </Button>
```